acquia_lift.module

```diff
@@ -0,0 +1,262 @@
+<?php
+
+/**
+ * @file acquia_lift.module
+ * Provides Acquia Lift Profiles integration.
+ */
+
+define('ACQUIA_LIFT_THUMBNAIL_WIDGET_SEPARATOR', '|');
+define('ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_DEFAULT', 'email');
+define('ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_IDENTIFIER', 'account');
+define('ACQUIA_LIFT_ENGAGEMENT_SCORE_DEFAULT', 1);
+define('ACQUIA_LIFT_GLOBAL_VALUE_DEFAULT', 1);
+define('ACQUIA_LIFT_ADMIN_SEPARATOR', '__');
+
+/**
+ * Implements hook_menu().
+ */
+function acquia_lift_menu() {
+  $items = array();
+  $items['admin/config/services/acquia_lift'] = array(
+    'type' => MENU_NORMAL_ITEM,
+    'title' => 'Acquia Lift',
+    'description' => 'Configuration settings for the Acquia Lift integration module.',
```

Comment:
I updated it here https://github.com/acquia/acquia_lift/pull/245/commits/c0f188b9f638dc248453dea92a05510e7f184ee4
This had been what was already in use in the D7 module (remember this was supposed to be a copy/paste story....).  We need to be a bit more consistent now when changes are made in one version to update both.  I believe the Drupal approach is to always start in the highest version and then backport so once this is out - we should update D8 and then backport as part of each Drupal lift 3 change.